contracts/core/foundry.toml

```diff
@@ -26,6 +26,7 @@ ffi = true
 
 # TODO: upgrade solc
 solc_version = "0.8.24"
+evm_version = "shanghai"
```

Comment:
Perhaps a comment explaining why we added this

Reply:
We originally added this because we needed to set the evm_version in order to interact with the BeaconBlockRoots contract locally. However, once applied, we found that the bytecode would change as the EVM was upgraded from the default version Paris, which is likely due to PUSH0 being activated in solc 0.8.24. After some deliberation, we decided to remove the BeaconBlockRoots test as we did not want the bytecode to change, and eliminated the evm_version config as well. External testing of this contract on Omega has shown it works, so we will leave it at that instead of committing to new bytecode.